pkg/controller/vsphere/util_test.go

```diff
@@ -67,4 +69,8 @@ func TestGetVSphereConfig(t *testing.T) {
 	if vSphereConfig.Global.Port != testPort {
 		t.Errorf("Expected zone %s, got %s", testZone, vSphereConfig.Global.Port)
 	}
+
+	if vSphereConfig.Global.InsecureFlag != testInsecureFlag {
+		t.Errorf("Expected zone %s, got %s", testZone, vSphereConfig.Global.Port)
```

Comment:
this error message is the same as the previous message, also it is reporting with the port. is this accurate or should it emit something about the insecure flag?

Reply:
oh, right :D ctrl+C, ctrl+V